app/components/utils/text-template/template.hbs

```diff
@@ -15,13 +15,14 @@
   as |model|
   }}
     {{model.label}}
+    {{!-- Disabled for the time being, because this makes for horrible UX (decision Johan Ronsse KAS-1171)
```

Comment:
Voor mij part mag dit weg.
We gaan dit nooit terugzetten op deze manier. (attach-popover > attach-tooltip).